src/bindings/python/tests/test_runtime/test_infer_request.py

```diff
@@ -493,6 +493,15 @@ def callback(request, _):
     infer_queue.wait_all()
 
 
+def test_infer_queue_userdata_is_empty(device):
+    core = Core()
+    param = ops.parameter([10])
+    model = Model(ops.relu(param), [param])
+    compiled_model = core.compile_model(model, device)
+    infer_queue = AsyncInferQueue(compiled_model, 1)
+    assert infer_queue.userdata == [None]
```

Comment:
Can we test for more values? Just bump up number of requests.

Reply:
added